internal/mode/static/manager.go

```diff
@@ -346,6 +352,18 @@ func registerControllers(
 		},
 	}
 
+	if cfg.EnableExperimentalFeatures {
+		backendTLSObjs := []ctlrCfg{
+			{
+				objectType: &gatewayv1alpha2.BackendTLSPolicy{},
+			},
+			{
+				objectType: &apiv1.ConfigMap{},
```

Comment:
It requires a bit of work, because we'll need to fetch the referenced ConfigMap from the API when we're using it, meaning we need to pass the client to the resolver. I think it's worth doing, but I think it should probably be a separate effort, and we should tackle Secrets at the same time using a similar approach. If we agree, I'll create a ticket for it 😄